src/modules/site-settings/edit-site-details.tsx

```diff
@@ -49,8 +43,15 @@ export default function EditSiteDetails( { currentWpVersion, onSave }: EditSiteD
 	const [ selectedPhpVersion, setSelectedPhpVersion ] = useState< AllowedPHPVersion >(
 		( selectedSite?.phpVersion as AllowedPHPVersion ) ?? DEFAULT_PHP_VERSION
 	);
-	const siteWpVersion = selectedSite?.wpVersion === 'latest' ? 'latest' : currentWpVersion;
-	const [ selectedWpVersion, setSelectedWpVersion ] = useState( siteWpVersion );
+	const getEffectiveWpVersion = useCallback(
```

Comment:
Good example of benefit for using `isWpAutoUpdating` - we don't need any more extra conditions, just this one.